astropy/modeling/tests/test_functional_models.py

```diff
@@ -153,6 +153,16 @@ def test_Shift_inverse():
     assert_allclose(m.inverse(m(6.789)), 6.789)
 
 
+def test_Scale_fittable():
+    m = models.Scale(1.2)
+    assert(m.fittable==True)
+
+
+def test_Shift_fittable():
+    m = models.Shift(1.2)
+    assert(m.fittable==True)
+
+
```

Comment:
I'd rather not have these tests. They don't add anything meaningful, just noise.